src/utils/transactions.ts

```diff
@@ -60,6 +60,15 @@ function getTxPendingTime(
   return timeLeftToForgeInMinutes > 0 ? timeLeftToForgeInMinutes : 0;
 }
 
+/**
+ * Formats a number of minutes to a string as "2h and 23m" or just "23m"
+ */
+function formatMinutes(minutes: number): string {
+  const remMin = Math.round(minutes % 60);
```

Comment:
I think that it's better `ceil` than `round` to not show 0 min when minutes are less than 0.5

Reply:
Done!